presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -1361,6 +1361,12 @@ public void testDistinctFrom()
         assertFunction("ARRAY ['puppies', 'kittens'] IS DISTINCT FROM ARRAY ['puppies', 'kittens']", BOOLEAN, false);
         assertFunction("ARRAY ['puppies', NULL] IS DISTINCT FROM ARRAY ['puppies', 'kittens']", BOOLEAN, true);
         assertFunction("ARRAY ['puppies', NULL] IS DISTINCT FROM ARRAY [NULL, 'kittens']", BOOLEAN, true);
+        assertFunction("ARRAY [1.23, 4.56] IS DISTINCT FROM ARRAY [1.23, 4.56]", BOOLEAN, false);
+        assertFunction("ARRAY [1.23, NULL] IS DISTINCT FROM ARRAY [1.23, 4.56]", BOOLEAN, true);
```

Comment:
Is the purpose to test all native container types? Then `boolean`, `Block` are still untested.

Also, why do we need this test and the next test?

Map distinct from also doesn't test all the native container types :)

Reply:
I added these tests to exercise delegation to decimal operators.

Reply:
@haozhun  In that case, do we also want to add tests for Map? Or just Array is good enough ? :)